airflow/cli/commands/remote_commands/config_command.py

```diff
@@ -64,3 +64,349 @@ def get_value(args):
         print(value)
     except AirflowConfigException:
         pass
+
+
+class ConfigChange:
```

Comment:
Not sure whether making it an `attr` class or `DataClass` would be better 🤔

Reply:
Should be a good idea to make it a dataclass (or attrs; this is simple enough either way should work just the same).

Reply:
Changed it